docs/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# CMakeLists for Freeciv21 Docs
+
+find_package(Sphinx REQUIRED)

```

Comment:
This makes Sphinx a hard dependency, which I don't think we want (and if we do, it should be documented). Remove the `REQUIRED` and wrap the rest into `if (Sphinx_FOUND)`?